src/hooks/fetch.ts

```diff
@@ -335,31 +335,43 @@ export class FetchInstrumentation {
   /**
    * Converts the headers object to a key-value pair object.
    * Fetch library uses multiple format to represent headers, this function will convert them all to a key-value pair object.
-   * @param {[string, string][] | Record<string, string> | Headers} headers Headers object as used by the fetch library
+   * @param {string[][] | Record<string, string | ReadonlyArray<string>> | Headers} headers Headers object as used by the fetch library
    * @returns {Record<string, string>} The headers as a key-value pair object
    * @private
    */
   private static convertHeadersToKeyValuePairs(
     // @ts-ignore
-    headers: [string, string][] | Record<string, string> | Headers
+    headers: string[][] | Record<string, string | ReadonlyArray<string>> | Headers
```

Comment:
Is the change from `[string, string][]` to `string[][]` also required from the Axios breaking change?
I think that  `[string, string][]` (an array of string pairs) is more precise than `string[][]` (an array of array of strings, that can technically also be `[['a'],['a'],['a']]` and not limited to pairs, so we better keep it if that's for us to decide

Reply:
@harelmo-lumigo 
This is a driven by udici-type changed
from
`export type HeadersInit = [string, string][] | Record<string, string> | Headers`
to
`export type HeadersInit = string[][] | Record<string, string | ReadonlyArray<string>> | Headers`